src/Plugin.php

```diff
@@ -78,4 +81,23 @@ public function wcBootstrap(Site $site, SingleSwitcher $siteSwitcher)
         add_action('woocommerce_new_product', [$wooCommerceGallery, 'saveGalleryIds']);
         add_action('woocommerce_update_product', [$wooCommerceGallery, 'saveGalleryIds']);
     }
+
+    public function acfBootstrap(Site $site, SiteSwitcher $siteSwitcher)
+    {
+        if (!\function_exists('get_field')) {
```

Comment:
@felly I know `get _field` is an ACF function but is there a constant or something  more at the top level of the plugin that we can use? What I'm worrying about is the fact that in some cases the file that define this function can be included manually and this plugin will try to bootstrap the integration with the ACF plugin. If there's something within the ACF main plugin file would be really cool, at the end if you require the plugin file you're bootstrapping the the plugin and there could be no problems.

Reply:
@widoz ACF defines a couple of constants - I will add a check for the constant `AFC` to the code.

Reply:
@felly I see the changes but still, why checking if ACF constants and `get_field`? Having a look at the main plugin file there's a function called `acf` see https://plugins.trac.wordpress.org/browser/advanced-custom-fields/trunk/acf.php#L657 I would just check against it. 

Another thing we might want to consider but this should be addressed on another issue is that, the acf plugin is loaded first because of the name, but in case of other plugins we might not be able to go this way. Maybe something to discuss with @bueltge about how to address dependecies like this where the dependency might loaded after MGM.

Reply:
We should avoid dependencies, the first rule in this case, because development is slow. But for this feature we should check it is ACF existent, so that we do not have any problems on loading. `acfBootstrap()` should only run if ACF is active, maybe via a Hook or a separate method that check them.

Reply:
@widoz @bueltge Thanks for your input!

I've rewritten the `acfBootstrap` call. This `acf/init` hook will be fired after ACF is initialized - as described in their [docs](https://www.advancedcustomfields.com/resources/acf-init/) / [code](https://plugins.trac.wordpress.org/browser/advanced-custom-fields/trunk/acf.php#L332):

```php
add_action('acf/init', function () use ($site, $singleSwitcher) {
    $this->acfBootstrap($site, $singleSwitcher);
});
```

I would love to see some similiar hook in your plugin:

```php
do_action( 'multisite-global-media/init', $site, $singleSwitcher);
```

Reply:
@felly Very good catch. I also agree the mgm should fire an event after initialization, I do no recall about the name convention for hook names but if you want please, open a PR, it would be highly appreciated.